### PR TITLE
Split large spellbook from UI enhancements setting

### PR DESCRIPTION
**Pull request by dydzio0614**
As in title - this setting is pretty large change that people will probably want separately from other "ui enhancements changes". Additional thing is that spellbook animation is broken for large spellbook.

### CODE REVIEW

**Comment by Laserlicht** [comment]
Is it possible to disable spellbook animation checkbox if the large spellbook is enabled?
To make it clear for the user that it isn't working with large spellbook?

**Comment by dydzio0614** [comment]
I do not think we support this particular case yet

**Comment by Laserlicht** [comment]
Okay, then maybe in the future. :)

File: config/schemas/settings.json [hunk 1]
```diff
@@ -131,6 +132,10 @@
 				"enableUiEnhancements" : {
 					"type": "boolean",
 					"default": true
+				},
+				"enableLargeSpellbook" : {
```
Comment (IvanSavenko) [inline config/schemas/settings.json:136]: Perhaps place this into "gameTweaks" category instead of "general"?

**Comment by IvanSavenko** [comment]
> I do not think we support this particular case yet (blocking option completely)

Checkboxes are buttons, so toggling button off and blocking it should work, e.g:
```cpp
    animationButton->setSelected(false);
    animationButton->block(true);
```

**Comment by Laserlicht** [comment]
Can we merge? :)

I want to update translation. ;)

**Pull request merged**